ci/build_common.sh

```diff
@@ -1,19 +1,52 @@
 #!/bin/bash
 
-set -euo pipefail
+set -eo pipefail
```

Comment:
Why did we have to remove `u`? I'd like to keep that if we can as I've often run into frustrating bugs caused by unset variables.

Reply:
nvm.